model-optimizer/mo/utils/broadcasting.py

```diff
@@ -124,7 +124,20 @@ def bi_directional_broadcasting(input_value: np.array, second_shape: np.array):
                                                                                                second_shape)
     return np.array(input_value * np.ones(second_shape), dtype=input_value.dtype)
 
+def expand_dims(input_value: np.array, axis):
+    """
+    This function contains implementation of numpy.expand_dims(numpy>=18.x)
+    """
+    if type(axis) not in (tuple, list):
+        axis = (axis,)
 
+    out_ndim = len(axis) + input_value.ndim
+    axis = np.core.numeric.normalize_axis_tuple(axis, out_ndim)
+    shape_it = iter(input_value.shape)
+    shape = [1 if ax in axis else next(shape_it) for ax in range(out_ndim)]
+
+    return input_value.reshape(shape)
+     
```

Comment:
As we discussed personally, to avoid code duplication I propose to remove these part which is copied from numpy and add multiple call of np.expand_dims inside `for` cycle.

Reply:
I deleted function expand_dims and added 'for' cycle instead of calling expand dims function